golang/pkg/cli/config_file.go

```diff
@@ -307,6 +308,13 @@ func DisabledServiceSettings(settings *Settings) *Settings {
 func PrintInfo(settings *Settings) {
 	log.Warn().Msg("🦩🦩🦩 Use the CLI tool only for NON-PRODUCTION purpose. 🦩🦩🦩")
 
+	if !settings.SettingsFile.DisableWelcomeMessage {
```

Comment:
Can we just make this happen once?

Reply:
Discussed in person. Output: too much chattiness is undesired by us and possibly by others as well.

Reply:
Fixed.